.github/workflows/continuous-integration.yml

```
@@ -0,0 +1,38 @@
+
+name: "Continuous Integration"
+
+on:
+  pull_request: null
+
+jobs:
+  static-analysis-psalm:
+    name: "Static Analysis with Psalm"
+    runs-on: ubuntu-18.04
```

```suggestion
    runs-on: ubuntu-latest
```

As previously discussed in other PRs :)

---

One PR is using latest and another one 18.04 for static analysis? Or is here a different reason than in doctrine/orm#8116 (or it was just forgotten :-D).

---

We settled for `latest`, just like in ORM, I am going to update this soon